packages/devtools_app/lib/src/screens/inspector/inspector_controller.dart

```diff
@@ -807,7 +805,7 @@ class InspectorController extends DisposableController
   }
 
   void selectionChanged() {
-    if (visibleToUser == false) {
+    if (!visibleToUser) {
```

Comment:
Fyi @pq. This is a lint fix from Dart code metrics that I'd be comfortable having in the recommended set if we were able to run Dart code metrics without significant performance overhead.

Reply:
Right, that's:

https://dartcodemetrics.dev/docs/rules/common/no-boolean-literal-compare

I'm a little surprised that was never defined in the core linter...